pages/common/composer.md

```diff
@@ -0,0 +1,31 @@
+# composer
+
+> Dependency Manager for PHP.
+
+- Interactively create a composer.json file:
```

Comment:
Since this is the first time referring to a composer.json file, defining it briefly would be a nice touch.